build.sbt

```diff
@@ -4,15 +4,14 @@ organization := "com.github.andyglow"
 
 scalaVersion := "2.11.12"
 
-crossScalaVersions := Seq("2.11.12", "2.12.4")
+crossScalaVersions := Seq("2.11.12", "2.12.4", "2.13.0")
 
 scalacOptions ++= Seq(
   "-encoding", "UTF-8",
   "-feature",
   "-unchecked",
   "-deprecation",
   "-Xlint",
-  "-Yno-adapted-args",
```

Comment:
This option is no longer supported by Scala 2.13, but -Xlint enables the warning.

Reply:
oh. let me look at this. there was a way to separate options needed for one scala version from another.

Reply:
i have just published `2.0.4` version with scala 2.13 support. will be available on maven soon. thanks for the contribution!

Reply:
Great thanks!